# sets the active manipulator

int32 robotid
string manip
---
